git/odb/storer.go

```diff
@@ -0,0 +1,16 @@
+package odb
+
+import "io"
+
+// Storer implements a storage engine for reading, writing, and creating
+// io.ReadWriters that can store information about loose objects
+type Storer interface {
```

Comment:
@technoweenie good call. I made the `Storer` and all of the concrete implementing types (`*FileStorer`, `*MemoryStorer`) private in c72121dc.